internal/graph/schema/alerts.graphqls

```diff
@@ -0,0 +1,197 @@
+extend type TeamEnvironment {
+	alerts(
+		"""
+		Get the first n items in the connection. This can be used in combination with the after parameter.
+		"""
+		first: Int
+
+		"""
+		Get items after this cursor.
+		"""
+		after: Cursor
+
+		"""
+		Get the last n items in the connection. This can be used in combination with the before parameter.
+		"""
+		last: Int
+
+		"""
+		Get items before this cursor.
+		"""
+		before: Cursor
+
+		"""
+		Ordering options for items returned from the connection.
+		"""
+		orderBy: AlertOrder
+
+		"""
+		Filter the returned objects
+		"""
+		filter: TeamAlertsFilter
+	): AlertConnection!
+}
+
+"""
+Alert interface.
+"""
+interface Alert implements Node {
+	"The unique identifier for the alert."
+	id: ID!
+	"The name of the alert."
+	name: String!
+
+	"The environment slug that the alert belongs to."
+	environmentName: String!
+
+	"The team slug that the alert belongs to."
+	teamSlug: Slug!
+
+	"The state of the alert."
+	state: AlertState!
+	"The query for the alert."
+	query: String!
+
+	"The duration for the alert."
+	duration: Float!
+
+	"The labels for the alert."
+	labels: [AlertKeyValue!]!
+
+	"The annotations for the alert."
+	annotations: [AlertKeyValue!]!
+}
+
+"""
+PrometheusAlert type
+"""
+type PrometheusAlert implements Node & Alert {
+	"The unique identifier for the alert."
+	id: ID!
+
+	"The name of the alert."
+	name: String!
+
+	"The environment slug that the alert belongs to."
+	environmentName: String!
+
+	"The team slug that the alert belongs to."
+	teamSlug: Slug!
+
+	"The state of the alert."
+	state: AlertState!
+
+	"The query for the alert."
+	query: String!
+
+	"The duration for the alert."
+	duration: Float!
+
+	"The labels for the alert."
+	labels: [AlertKeyValue!]!
+
+	"The annotations for the alert."
+	annotations: [AlertKeyValue!]!
+
+	"The prometheus rule group for the alert."
+	ruleGroup: String!
```

Comment:
Har dette verdi for andre, eller er det litt random hva gruppenavnet blir?

Reply:
Det er navnet på regelgruppen, se feks her: https://prometheus.dev-gcp.nav.cloud.nais.io/alerts

Den øverste har da ruleGroup "namespace aap" for reglene som ligger under. Lar den foreløpig stå.

Reply:
Men er disse noe de har kontroll på selv?